### PR TITLE
0.9 release staging

### PR DESCRIPTION
**Pull request by borgmanJeremy**

### CODE REVIEW

File: packaging/debian/changelog [hunk 1]
```diff
@@ -1,5 +1,5 @@
-flameshot (0.8.5-1) unstable; urgency=medium
```
Comment (hosiet) [inline packaging/debian/changelog:1]: I guess this is a typo.

**Review by mmahmoudian — COMMENTED** [review]
I marked Some typos.

**Comment by hosiet** [comment]
Besides, I'm not sure why the previous v0.8 release note was discarded. If the old release note is discarded, it seems that it can be found nowhere unless someone goes into git history to pull it out.

Since this is changelog-like information, the best way might be adding v0.9 changelog before old v0.8 changelog text.

**Comment by mmahmoudian** [comment]
I agree with @hosiet

File: docs/ReleaseNotes.md [hunk 2]
```diff
@@ -1,63 +1,40 @@
...
+- New global shortcut menu. All actions hotkeys are fully customizable. 
+  
+- Ability to take "symmetric" selections by holding down the Shift key while resizing the selection. 
+
+- The rectangle tool will now round the corners of the rectangles based on the current thickness
+
+- All imgur uploads are now tracked in the "Upload History" menu. This makes it much easier to delete of images off imgur
+or find the upload link later.
+
+- Added "check for new release" feature. This allows users on MacOs / Windows / and AppImages to easily check for updated versions.
```
Comment (ZetaoYang) [inline docs/ReleaseNotes.md:30]: MacOs -> MacOS

**Review by ZetaoYang — COMMENTED** [review]
I agree with @mmahmoudian. The release notes for previous versions should be retained.

**Comment by ZetaoYang** [comment]
Besides, `docs/dev/package.md` and `/docs/dev/release.md`. These two documents are outdated and I recommend removing them.

**Comment by borgmanJeremy** [comment]
Thanks everyone, Ill get this feedback cleaned up and then get the release out. Do you think its worth releasing this and RC1 or should I just release it as 0.9 and any issues that come up can be 0.9.x?

**Comment by mmahmoudian** [comment]
Having it as Release Candidate is a good idea, but I don't know Joe many people will be out beta tester. Of course we ourselves can try it out and make sure there is no major issues, as the end user would go for alternative software if they face major bugs. Also this would help us in not getting flooded with similar bug reports.

File: packaging/debian/changelog (hunk 1 above)
Comment (borgmanJeremy) [reply packaging/debian/changelog:1]: Yeah, it should read 0.9.0-1, Fixed. Thanks

**Pull request merged**